### DISCUSSION TITLE
Is it possible to include Deepin-Extra packages in Fedora DDE 20.2?

### DISCUSSION THREAD

**Discussion by serial02** [linuxdeepin/developer-center#3218] [Features Request & Ideas | 特性请求 & 头脑风暴]
Hello everybody! 

 It is possible to include Deepin-Extra packages in Fedora DDE 20.2, as all applications are working fine in ArchLinux DDE 20.2. 

 Note: Deepin-Extra packages were installed after installing DDE 20.2 by Archinstall 220601.

 By the way, the Fedora DDE 20.2 login screen is a blast!  Congratulations!

![Captura de tela de 2022-06-03 15-48-14](https://user-images.githubusercontent.com/32621790/171934268-38c3ba41-ea55-44fb-b8f8-ad5587d81740.png)

**Comment by BLumia** [comment]
It's possible but it depends on the DDE-related package maintainer of the distro you're using.

**Comment by cheese** [comment]
It is lack of human power for further enhancement of DDE in Fedora.

**Reply by serial02** [reply]
> It is lack of human power for further enhancement of DDE in Fedora.

You and the whole team are doing a great job!  I just have to say thank you! @cheese 

I would love to have in-depth knowledge to help in this endeavor.
 If I could help financially, I wouldn't hesitate.

 Anyway, everyone helps in what they can.  After all, we can't offer anything more than what we have at the moment.

 I'm using this "Spin" Fedora 36 with DDE 20.2 in VM, waiting for improvements and updates, sending my impressions and suggestions.  After all, everyone is a little technical and crazy.....lol.

 Of all the DDEs, this is the one that fascinates me the most. 

 Why do I say this? 

 I usually say that:
 Archlinux is the dark side of the force.
 Debian or .deb is the soft side of the force.
 Fedora or rpm is the balanced side of the force.  It has everything up to date with no "system" breaks.  It's very consistent for everyday use, from Official Distros to Spins.

 As for Deepin 20.6, which I admire a lot, it has yet to find its way and is almost there, among the best distributions.

 What do I like about Deepin 20.6? 

 A new distro that emerged in the Linux world showing exemplary beauty leaving many others in the "slipper" (Brazilian term).

 What I don't like about Deepin 20.6
 Outdated store, confusing app search by terminal, many apps I like to use newer only exist via flatpak, where some for having outdated flatpak base cannot be installed or or errors occur after installation which can be seen here 
(https://github.com/linuxdeepin/developer-center/discussions/3216).

 Success to all!

**Reply by cheese** [reply]
DDE in Fedora is not in a good shape. The original packager of DDE in Fedora quitted a few years ago. I don't want DDE to be retired(deleted) from Fedora, but what I can afford is only keeping it built and started without a reproducible crash. And to make the maintainment burden minimal, I just keep the existing Deepin packages in Fedora sychronized with the ones in Arch Linux, without intruducing new ones.

If you want to help DDE in Fedora directly, just learn RPM packaging and join the Fedora packager team.

For financial support, there is no exsting way. May be you can:

1. Donate to the Deepin upstream and let them employ a full-time developer to work on DDE in Fedora.
2. Donate to the Deepin upstream and let the employees working on packages for openEuler also maintain the ones in Fedora. openEuler is a China-based Linux ditro and also RPM-based.
3. Donate a case-by-case bonus. For example, fixing one DDE issue in Red Hat Bugzilla can get a bonus of 1000 CNY or 150 USD.
